src/command_helpers/asset_helper.rb

```diff
@@ -46,6 +46,13 @@ def raise_error_if_node_is_missing
           Can not find node: #{options.node}
         EOF
       end
+
+      def error_if_asset_file_doesnt_exist(asset_name, asset_path)
+        return if File.exist?(asset_path)
+        raise InvalidInput, <<-EOF.squish
+          The "#{asset_name}" asset does not yet exist
```

Comment:
Remove the `yet` from this comment. It makes it sound like, the asset should be added, which doesn't make sense if we are trying to delete it.